README.md

```diff
@@ -39,12 +46,26 @@ This documentation is written as of 06/17/2014. JIRA 6.3 implements OAuth1. Most
     # Now, create the public key associated with that private key
     openssl rsa -in mykey.pem -pubout
     ```
-2. Generate a consumer key. You can use python uuid.uuid4() to do this.
+2. Generate a consumer key. You can use python uuid.uuid4() to do this, for example:
+
+    ```
+    $ python
+    Python 2.7.10 (default, Jul 30 2016, 19:40:32)
+    [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> import uuid
+    >>> print uuid.uuid4()
+    210660f1-ca8a-40d5-a6ee-295ccbf3074d
+    >>>
+    ```
+    
 3. Configure JIRA for external access:
      * Go to AppLinks section of your JIRA - https://JIRA_SERVER/plugins/servlet/applinks/listApplicationLinks
      * Create a Generic Application with some fake URL
      * Click Edit, hit IncomingAuthentication. Plug in the consumer key and RSA public key you generated.
-4. Get access token using this [script](https://github.com/lakshmi-kannan/jira-oauth-access-token-generator/blob/master/generate_access_token.py). These are the ones that are printed at the last. Save these keys somewhere safe. 
+4. Get access token using this [script](https://github.com/lakshmi-kannan/jira-oauth-access-token-generator/blob/master/generate_access_token.py). You may need to install additional libraries to run that script, and you
```

Comment:
We should move this script into this repository in `etc/` directory.

Anyway not a big deal for now.